meilisearch-release-changelog/v1.4.0.md

```diff
@@ -0,0 +1,89 @@
+# v1.4.0 release changelogs
+
+<!-- The following line should NOT be put in the official release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Is something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+Meilisearch v1.4 introduces features to improve the relevancy of the Meilisearch results by adding three news settings: `nonSeparatorTokens`, `separatorTokens` and `dictionary`.
+
+<!-- The following lines should ONLY be put in the official release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
+
+Some SDKs might not include all new features—consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+# New features and improvements 🔥
+
+## Customize text separators
+
+The Meilisearch word segmentation uses a default list of separators to segment words. This default list might not sometimes fit your use case: the `nonSeparatorTokens`/`separatorTokens`` setting allows you to remove/add some tokens from/to the default list of separators.
+
+To remove tokens:
+
+```bash
+curl \
+  -X PUT 'http://localhost:7700/indexes/articles/settings/non-separator-tokens' \
+  -H 'Content-Type: application/json'  \
+  --data-binary '["@", "#", "&"]'
+```
+
+To add tokens:
+```bash
+curl \
+  -X PUT 'http://localhost:7700/indexes/articles/settings/separator-tokens' \
+  -H 'Content-Type: application/json'  \
+  --data-binary '["&sect;", "&sep"]'
+```
```

Comment:
I find it a bit easier to understand this if we begin with the setting with the simpler name (`separator-tokens`), then progress to the more complex name (`*non*-separator-tokens`)

````suggestion
Add a character to the `separator-tokens` list to use it as a word separator:

```bash
curl \
  -X PUT 'http://localhost:7700/indexes/articles/settings/separator-tokens' \
  -H 'Content-Type: application/json'  \
  --data-binary '["&sect;", "&sep"]'
```

Add a character to the `non-separator-tokens` list when you don't want Meilisearch to use it to separate words:

```bash
curl \
  -X PUT 'http://localhost:7700/indexes/articles/settings/non-separator-tokens' \
  -H 'Content-Type: application/json'  \
  --data-binary '["@", "#", "&"]'
```
````